### PR TITLE
chat-cli: mvp graph-store migration

### PR DESCRIPTION
**Pull request by Fang-**
Makes chat-cli listen to graph-store rather than chat-store.

Along the way, cuts some features. No longer supports channel creation,
deletion, permission management or inviting. No longer lets you send
messages to multiple targets at once.

No longer displays messages from all joined chats, and no longer lets
you join or leave chats. Instead, chat-cli solely concerns itself with
viewing chat content you're already subscribed to (through use of
Landscape or other rich clients). `;view ~host/chat` to start printing
messages for that chat when they come in, `;flee` to toggle off. `;view` to
see what chats are enabled, `;chats` to see all available chats.

The removal of features acknowledges chat-cli's lack of real support for
the modern, group-centric state of userspace.
The intent of `;view` ux is to prepare chat-cli for a multi-session
terminal world.

~~Opening this PR in case it needs to go in asap, but will try and find some time in the next 24 hours to add actual multi-session support to this.~~ Multi-session support is in.

The approach here is one of "minimum viable migration". The code style is (or rather, remains) a bit rough around the edges, but fixing that is not in scope here.

Tested this a bit by playing around with graph generators, think it's probably good, but want to test again once @liam-fitzgerald's frontend changes are in just to make sure this plays nice in "wilder" contexts. Message rendering should work fine, but might look a tad janky.
(I assume an updated `/mar/graph-validator/chat.hoon`, with `notification-kind:grow` arm, will find its way into `next-userspace`.)

cc @syzygyzer for updating the messaging docs, though maybe we just want to remove chat-cli from the docs altogether for now.

### CODE REVIEW

File: pkg/arvo/app/chat-cli.hoon
Comment (Fang-) [inline pkg/arvo/app/chat-cli.hoon:777]: idk what's up with this, by the way. Nothing's shadowing `store`, nothing about this even changed, but it just won't resolve to the right thing. What dumb mistake could I be overlooking?

**Comment by matildepark** [comment]
Thank you for your work @Fang-

**Comment by matildepark** [comment]
Pointed at the current Chat -> graph-store migration location. @liam-fitzgerald can you review this?

**Review by liam-fitzgerald — APPROVED** [review]
This looks good! Pushed a couple of changes for things I found while testing, other than that LGTM!

**Comment by ohAitch** [comment]
Is there at least still support for generators to manage channel state?
"Urbit is unusable without a web browser" feels deeply weird.

**Comment by matildepark** [comment]
You haven't been able to have complete parity with Landscape's web front-end for quite a while now — since around OS1, I think. 

`:chat-cli` is from a point in time where we wrote IRC and left it at that; Landscape is a suite of Gall agents centred around the group pattern with an accompanying front-end, whose scope we've contained to the one web client as a team. Anything else is essentially a bonus until Landscape gets assigned a CLI maintainer. (Not to mention CLI parity is blocked on quite a few things to actually provide that experience. That said, @Fang- spends quite a bit of time these days working on the CLI experience, so I expect more improvements to come here.)

**Pull request merged**
